docs-src/mkdocs.yml

```diff
@@ -39,9 +39,17 @@ pages:
     - Secure Shell (SSH): troubleshooting/ssh.md
 
 site_dir: ../docs
+
 theme:
-    name: 'material'
-    custom_dir: 'material'   
+  name: 'material'
+  palette:
+    primary: 'indigo'
+    accent: 'indigo'
+
+extra:
+    version: v0.10.3
+    logo: 'static/logo_icon.png'
+    author: xfra
```

Comment:
So here, the `theme` block and the `extra` block changed with the new version of mkdocs (I'm using mkdocs v0.17.2)

I think the newer version of mkdocs did away with the `extra` block. Some stuff that you have here in the `extra` block actually needs to go in the `theme` block. What happens when I build this is our color scheme isn't preserved.

This is how I have the `theme` block:

```
 theme:
     name: 'material'
     custom_dir: 'material'
     palette:
         primary: 'red'
         accent:  'grey'
     font:
         text: 'Roboto'
         code: 'Roboto Mono'
     logo: 'static/logo_icon.png'
```

Give this a try and you should see the color scheme align with what is currently available at layer0.ims.io.

Reply:
Perfect! I have that version of mkdocs too. I'll update my pr